persistence/mapper/src/main/scala/net/liftweb/mapper/ProtoExtendedSession.scala

```diff
@@ -59,6 +59,13 @@ trait UserIdAsString {
   def userIdAsString: String
 }
 
+/**
+ * The root trait for defining the session cookie path for extended sessions
+ * that defines the default session cookie path: "/".
+ *
+ * Extend this trait then mix it into an extended session singleton to change
```

Comment:
Do you ever need to extend this trait explicitly? Can you not just change this directly in your extended session singleton?

Reply:
You could directly make the change, so long as you only wanted to make the change to a single instance. Which, I suppose given how these are used, that's all you'd do. I'll remove the remark.